time-elements.js

```diff
@@ -587,6 +589,33 @@
     }
   }
 
+  // Private: Format a time according to the `timezone` attribute values.
+  //
+  // el - The local-time element to format.
+  //
+  // Returns a timezone String or null if no timezone formats are provided.
+  function formatTimezone(el) {
+    // retrieve format settings from attributes
+    var options = {
+      timezone: el.getAttribute('timezone')
+    };
+
+    // remove unset format attributes
+    for (var opt in options) {
+      if (!options[opt]) {
+        delete options[opt];
```

Comment:
This whole `options` hash dance could be simply replaced with:

```js
var timezoneSetting = el.getAttribute('timezone')
if (!timezoneSetting) return
var tzDisplay = ...
return tzDisplay
```

No `options` hash, looping over its keys, `Object.keys()` check. There is always ever one key (for now).